examples/linux/policyfile_cookbook/chefignore

```diff
@@ -0,0 +1,108 @@
+# Put files/directories that should be ignored in this file when uploading
```

Comment:
Do not include a chefignore, as this won't be uploaded to a chef server

Reply:
See comment below about this being an example cookbook. In hindsight I should have responded here.

Reply:
I see your point here and retract my comment for now